### DISCUSSION TITLE
Ship temperature variable

### DISCUSSION THREAD

**Discussion by Pankratyev** [EDCD/EDDI#2659] [EDDI Standalone Q&A] [answered]
Hi guys.
I can't find a variable containing the current temperature of the ship. I want EDDI to tell me when the temperature reaches a certain value.
Does such a variable exist? Сan someone help me with this?

**Comment by Tkael** [comment, marked as answer]
This information is not available from the information that Frontier shares with 3rd party devs unfortunately.

**Reply by Pankratyev** [reply]
That's what I thought, but I hoped I was wrong. Thanks a lot for the reply!
